src/main/java/org/folio/rest/impl/OaiPmhImpl.java

```diff
@@ -91,6 +91,7 @@ public void getOaiRecords(String verb, String identifier, String resumptionToken
     String generatedRequestId = UUID.randomUUID().toString();
     RepositoryConfigurationUtil.loadConfiguration(okapiHeaders, generatedRequestId)
       .onSuccess(v -> {
+        String requestId = generatedRequestId;
```

Comment:
What is the purpose of this variable?

Reply:
It is needed to remove config from map in case of exception catching at line 152.